### PR TITLE
Release notes for 6.1.3

### PR DESCRIPTION
**Pull request by andrewvc**
Does what it says on the tin.

Diff: `git log v6.1.2..ff7c29c`

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -13,6 +14,46 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-3]]
+=== Logstash 6.1.2 Release Notes
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: I think this should be `6.1.3`

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -13,6 +14,46 @@ See also:
...
+
+* Fix bug where with terminating input plugins in-memory queue might not be drained. This could happen in some situations with inputs like the stdin input or the Elasticsearch input. This could result in some messages not being processed.
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: May want to link to the bug.

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -13,6 +14,46 @@ See also:
...
+*Elasticsearch Filter*
+
+* If elasticsearch response contains a shard failure, then tag_on_failure tags are added to Logstash event
+* Enhancement : add support for nested fields
+* Enhancement : add 'docinfo_fields' option
+* Enhancement : add 'aggregation_fields' option
+
+*Elasticsearch Input*
+
+* Add support for scheduling periodic execution of the query #81
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: May want to hyper link ES input #81 (or remove the number)

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -13,6 +14,46 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-3]]
+=== Logstash 6.1.2 Release Notes
+
+* Fix bug where with terminating input plugins in-memory queue might not be drained. This could happen in some situations with inputs like the stdin input or the Elasticsearch input. This could result in some messages not being processed.
+
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: May want to call out Windows: correctly handle paths with spaces #8931

**Comment by jakelandis** [comment]
Some minor comments to address at your discretion. 

LGTM pending the correction to the header version `6.1.3`

File: docs/static/releasenotes.asciidoc (hunk 2 above)
Comment (andrewvc) [reply docs/static/releasenotes.asciidoc]: I would, but this is more user facing, and https://github.com/elastic/logstash/pull/9024 is quite technical. Basically garbage for the average user. I'm glad to improve the wording here however.

File: docs/static/releasenotes.asciidoc (hunk 4 above)
Comment (andrewvc) [reply docs/static/releasenotes.asciidoc]: Great call, not sure how I missed that.

**Pull request closed without merging**